python/across_api/across/api.py

```diff
@@ -0,0 +1,51 @@
+from typing import Annotated, Optional
+
+from fastapi import Depends, Query
+
+from ..base.api import app
+from .hello import Hello
+from .resolve import Resolve
+from .schema import HelloSchema, ResolveSchema
+
+
+# Depends functions for FastAPI calls.
+async def name(
+    name: Annotated[
+        str, Query(description="Name of astronomical object to convert to coordinates.")
+    ]
+) -> str:
+    return name
+
+
+async def your_name(
+    name: Annotated[
+        Optional[str],
+        Query(
+            description="Name of person to greet.",
+            title="Your Name",
+        ),
+    ] = None
+) -> Optional[str]:
+    return name
+
+
+SourceNameDep = Annotated[str, Depends(name)]
+YourNameDep = Annotated[Optional[str], Depends(your_name)]
+
+
+# API End points
+@app.get("/")
+async def hello(name: YourNameDep) -> HelloSchema:
+    """
+    This function returns a JSON response with a greeting message and an optional name parameter.
+    If the name parameter is provided, the greeting message will include the name.
+    """
+    return Hello(name=name).schema
+
+
+@app.get("/ACROSS/Resolve")
```

Comment:
It's a bit unusual to have uppercase characters in URLs.

Reply:
Fixed in 893f9c1.